.changeset/many-forks-sniff.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/kit': patch
+---
+
+[breaking] implement new layout system
```

Comment:
Do you plan to update the PR description or add a link to the docs explaining the change? Users combing changelogs to see what they have to update would appreciate

Reply:
Yes I will write up some migration comments in the PR description, which should probably noted in the description
```suggestion
[breaking] implement new layout system (see the PR for migration instructions)
```